setup.py

```diff
@@ -2,36 +2,43 @@
 
 from setuptools import setup, find_packages
 
-install_requires = [
+INSTALL_REQUIRES = [
     'lxml',
     'setuptools',
-    'Django>=1.4',
+    'Django>=1.4,<1.7',
     'django-appconf',
     'django-cms>=2.4',
     'django-classy-tags>=0.3.2',
-    'django-haystack>=2.0.0',
+    'django-haystack>=2.4.0.dev0',
```

Comment:
We can't aggressively require latest haystack or specially a dev version.
Most projects use 2.1 and first need to go through upgrade process to reach 2.3 which is latest stable.
So I don't approve of this, instead I would add support to handle in case this DoNotIndex exception does not exist.